test/test_mpi.jl

```diff
@@ -3,8 +3,6 @@ module TestMPI
 using Test
 using TrixiAtmo
 
```

Comment:
```suggestion
using Trixi: Trixi

```
We need to use Trixi for `Trixi.mpi_isroot()` (could alternatively be used from TrixiBase.jl, but that should be the same), `Trixi.MPI.Barrier`, and `Trixi.mpi_comm()`.